index.html

```diff
@@ -102,5 +102,27 @@ <h2>Heading</h2>
             r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
             ga('create','UA-XXXXX-X','auto');ga('send','pageview');
         </script>
+        
+        <!-- Add font awesome icons -->
```

Comment:
I believe if we wrap this in a div with an id or container, and style it with margin 0 auto we can get the icons more center.